app/config.py

```diff
@@ -4,11 +4,12 @@
 
 class AppConfig(BaseSettings):
     model_config = SettingsConfigDict()
+    python_env: str | None = None
     host: str | None = None
     port: int | None = None
     log_config: str | None = None
     mongo_uri: str | None = None
-    mongo_database: str = "cdp-python-backend-template"
+    mongo_database: str = "python-mcp-server-demo"
```

Comment:
could we revert this back, when we generate a new project from the template we replace cdp-python-backed-template with the service name (which should map to their db name)